docs/docs/persistent-volumes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Stateful Applications Using Local Persistent Volumes
+---
+
+Marathon applications lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can create a stateful application by specifying a local persistent volume.
+
+When you specify a local volume or volumes, tasks and their associated data are “pinned” to the node they are first launched on and will be relaunched on that node if they terminate. Local volumes enable stateful tasks because the task can be restarted without data loss.
+
+## Scaling your app
+
+When you scale your app instances/tasks down, the volumes associated with the destroyed tasks/app instances are detached. At this point, you may destroy the volume from the Marathon web interface. You may wish to leave it in the Detached state, however, if you think you will be scaling up your app again; the data on the volume will still be there. **Note:** If your app is deleted, any associated volumes will also be deleted.
```

Comment:
I'd go for `instances` instead of `instances/tasks`. Do we have a settled terminology on this? afaik the UI says `task` while the config is about `instances`.